backend/src/appointment/controller/calendar.py

```diff
@@ -501,7 +516,7 @@ def create_vevent(
             slot.start.replace(tzinfo=timezone.utc) + timedelta(minutes=slot.duration),
         )
         event.add('dtstamp', datetime.now(UTC))
-        event.add('status', 'CONFIRMED')
+        event.add('status', 'NEEDS-ACTION' if on_hold else 'CONFIRMED')
```

Comment:
This needs to be tested directly in Thunderbird.

Reply:
Whoops, this was supposed to be `TENTATIVE`, but we can keep `CONFIRMED` as well. I don't think this is needed for the prompt to update. According to RFC 5546 and [Philipp](https://github.com/thunderbird/appointment/issues/459#issuecomment-2491213535), there needs to be a METHOD:REQUEST. I'll add that too.